R/fb_audience.R

```diff
@@ -212,3 +212,10 @@ fbad_create_lookalike_audience <- function(fbacc, name, origin_audience_id, rati
     fromJSON(res)$id
 
 }
+
+#' List all Custom Audiences for Ad account
+#' @inheritParams fbad_list_ad
```

Comment:
Regarding your "How can I remove the 'statuses' parameter since it's not supported by /customaudiences?" question from the PR description, I think the documentation should be actually written for this function and not simply taking over the `fbad_list_ad` docs via `inheritParams`, so this could become something like:

```r
#' @inheritParams fbad_request
#' @param fields character vector of fields to get from the API, defaults to \code{id}. Please refer to the Facebook documentation for a list of possible values.
```

And if we want to support the other params, like business_id or filtering, we might have to tweak `fbad_list_ad` a bit after all. But for now, I think that's not required and a basic list of custom audiences might be enough.